### PR TITLE
Sprint

### PR DESCRIPTION
**Pull request by ipdemes**

### CODE REVIEW

**Comment by ipdemes** [comment]
@junghans , does this build fail because of the code coverage test?

**Comment by junghans** [comment]
Yes, you will need to add doxygen documentation for your new classes.

File: examples/lax_wendroff/lax_wendroff.cc [hunk 1]
```diff
@@ -0,0 +1,62 @@
+/*~-------------------------------------------------------------------------~~*
+ * Copyright (c) 2014 Los Alamos National Security, LLC
```
Comment (junghans) [inline examples/lax_wendroff/lax_wendroff.cc]: 2017?

Comment (ipdemes) [inline examples/lax_wendroff/lax_wendroff.cc]: We should replace header with the FleCSI one, I guess

**Comment by ipdemes** [comment]
@junghans , can I merge?

**Comment by junghans** [comment]
Sure!

**Comment by tuxfan** [comment]
We have a tool in the cinch command-line interface (ccli) that can do all of the files at once. I’ll assign a task to myself to take care of this.

--
Dr.-Ing. Benjamin Karl Bergen, M.Sc.

Los Alamos National Laboratory
P.O. Box 1663, MS B287
Los Alamos, NM 87545

work: (505) 667-1699
cell: (505) 412-7118

**Pull request merged**